scipy/optimize/setup.py

```diff
@@ -80,6 +80,21 @@ def configuration(parent_package='',top_path=None):
 
     config.add_subpackage('_trustregion_constr')
 
+    # cython optimize API for zeros functions
+    cython_optimize_tests = os.path.join('cython_optimize', 'tests')
+    config.add_subpackage('cython_optimize')
+    config.add_subpackage('cython_optimize.tests')
+    config.add_data_files('cython_optimize.pxd')
+    config.add_data_files(os.path.join('cython_optimize', '*.pxd'))
```

Comment:
If we don't have `cython_optimize.zeros` public, we should be able to avoid this I guess. There's a bit more `add_data_files` going on here than I'd like.

Reply:
✔️ done